app/models/experience.rb

```diff
@@ -12,6 +12,7 @@ class Experience < ApplicationRecord
   geocoded_by :address
   after_validation :geocode, if: :will_save_change_to_address?
   # TODO: The seeds don't have proper addresses
+  scope :active, -> { where(status: 'active') }
```

Comment:
Should probably use an enum for this, as it will create the scope as well as other methods like `active?`, `active!`, `building?`, etc.
```suggestion
  enum status: { building: 'building', active: 'active' }
```